### DISCUSSION TITLE
[Request] Configuration for energy storage/transfer of Adjustable SU and IDSU

### DISCUSSION THREAD

**Discussion by chimericdream** [TechReborn/TechReborn#3134] [Ideas]
Given how expensive they are, it would be nice if both the ASU and IDSU were capable of transferring more than 8.1k energy. For players on my personal server, we would especially like to modify the IDSU to be able to transmit at least an order of magnitude more energy per tick, if not two.

I know that there are configs for many other things, especially when it comes to energy stored within the blocks. What are the chances we could get a similar config for energy transfer out of those same blocks?

Thanks!

**Comment by KEJWII** [comment]
I support the idea
but it is strange that they cannot be configured as you can change almost all generators, machines etc in config file but not battery blocks

although it would be useful to have a battery multiblock, which would be a great solution for storing a lot of power
